region/brinstar/pink/Big Pink.json

```diff
@@ -1798,7 +2151,8 @@
       "note": [
         "While using multiple Power Bomb boost Spring Ball jumps, to climb the room,",
         "overload PLMs by moving through the camera scroll blocks in front of the passageway leading to the exit of Spore Spawn Supers.",
-        "Note that one Power Bomb can be saved by carefully jumping out of this morph passageway.",
+        "Note that one Power Bomb can be saved with a Spring Fling; pause and just before the screen is fully black, jump, disable Spring ball, then buffer right on the unpause.",
```

Comment:
If you are using the item pedestal and in direct, this would force collection of the item.